spec/integration/inventory2_spec.rb

```diff
@@ -239,11 +240,7 @@ def fact_plan(name = 'facts_test')
           },
           {
             name: 'bar',
-            nodes: [
-              {
-                name: 'bar_1'
-              }
-            ],
+            targets: [],
```

Comment:
The test adds a string node `bar_1` to the `bar` group.  By having it already present in the group it seems like it wasn't really testing what was expected.

Reply:
I think I just did a bad job describing/writing that test. What that **should** be testing is *This should do nothing if the user adds the target to a group it is already a part of.* [BOLT-942](https://tickets.puppetlabs.com/browse/BOLT-942). So, given `bar_1` in group `bar` adding a target `bar_1` does not override, or duplicate `bar_1`. Though I guess the test as is written does not really prove that the `bar_1` defined in inventory has been overridden. 

This test could certainly be improved. (I see an unused variable in the plan, plus the confusing/wrong descriptions). I can work on that as a maint PR against the inventory v1 integration test and then port those changes over as separate work.

Reply:
https://github.com/puppetlabs/bolt/pull/958